vagga.yaml

```diff
@@ -25,9 +25,9 @@ containers:
         dependencies: true
       - !Install
         - libpq
-      - !Py3Requirements requirements/main.txt
+      - !Py3Requirements requirements/production.txt
```

Comment:
This container need requirements from production container. We run it with whitnoise.
This was my fault before